www/source/partials/docs/_using-hab-continuous-deployment.html.md.erb

```diff
@@ -0,0 +1,56 @@
+# <a name="continuous-deployment" id="continuous-deployment" data-magellan-target="continuous-deployment">Continuous Deployment Using Channels</a>
+
+Continuous deployment is a well-known software development practice of building and testing code changes in preparation for a release to a production environment. 
+
+<%= partial "/partials/global/channel-overview" %>
+
+By default, every new package is placed in the `unstable` channel by Builder.  Packages in the `unstable` channel cannot be started or installed unless you specify the `--channel` flag in the `hab` CLI, or set the `HAB_BLDR_CHANNEL` environment variable to a non-stable channel. This is because the default channel used by the `hab` CLI when starting, installing, or loading packages is the `stable` channel. The `stable` channel indicates a level of stability and functionality suitable for use in multi-service applications or as a dependency for your service.
+
+To promote your package to a channel, you must first upload it to Builder and then use the `hab pkg promote` subcommand to promote the package to the intended channel. To combine operations, the `hab` CLI allows you to do both in one command by using the `--channel` option when uploading your package. The following shows how to upload and promote a package to a custom channel named `test`.
```

Comment:
You can also just have Builder build it for you, which will place it into the unstable channel automatically, and you can promote from there.